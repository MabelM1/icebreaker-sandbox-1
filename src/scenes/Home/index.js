import React from 'react'
import Title from 'components/Title'

const Home = () => (
  <div>
    <Title>Home</Title>
  </div>
)

export default Home
