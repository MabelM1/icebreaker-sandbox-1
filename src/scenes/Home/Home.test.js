import React from 'react'
import renderer from 'react-test-renderer'
import Home from './'

describe('<Home />', () => {
  it('renders initial UI', () => {
    const tree = renderer.create(<Home />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
