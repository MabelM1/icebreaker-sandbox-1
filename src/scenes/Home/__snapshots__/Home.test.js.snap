// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> renders initial UI 1`] = `
<div>
  <h1
    className="eBQaql"
  >
    Home
  </h1>
</div>
`;
